### ISSUE TITLE
Option not to save headers, method, content as binary?

### ISSUE THREAD

**Issue by jeromegn** [oesmith/puffing-billy#59]
I noticed everything is being saved as binary. This is really hard to edit. Sometimes you make requests on an API and want to control the response for multiple test cases.

I think I might understand the value in using binary for the content, but the url, headers and method should be plain text. This is just obscuring very useful data for debugging.

If content wasn't force encoded to binary, JSON responses could be modified ever so slightly to test edge cases more difficult to get through a real request.

Basically, I'd like better control over the cached requests / responses.

**Comment by ronwsmith** [comment]
@jeromegn, just checking in on this issue.  Let me know once you've tried my suggestions above.

**Comment by jeromegn** [comment]
Interesting.

I must say I haven't used the gem in a months and I'm not sure I will given how our stack has evolved. i hope this thread helps somebody else though!

Thanks!
